hack/common.sh

```diff
@@ -41,24 +41,29 @@ readonly OS_SCRATCH_IMAGE_COMPILE_TARGETS=(
   examples/deployment
 )
 readonly OS_IMAGE_COMPILE_BINARIES=("${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]##*/}" "${OS_IMAGE_COMPILE_TARGETS[@]##*/}")
-
 readonly OS_CROSS_COMPILE_PLATFORMS=(
   linux/amd64
   darwin/amd64
   windows/amd64
   linux/386
 )
-readonly OS_CROSS_COMPILE_TARGETS=(
+
+if [ "_$@" = "_test/e2e/e2e.test" ]; then
```

Comment:
Im neutral.
- I thought complete orthogonality was the goal 
- but I really am open to whatever. 
I think Jordan has an exact opinion..?